udapi/core/root.py

```diff
@@ -82,6 +84,20 @@ def zone(self, zone):
     def bundle(self):
         return self._bundle
 
+    @bundle.setter
+    def bundle(self, bundle):
+        self._bundle = bundle
+
+    @property
+    def children(self):
+        logging.debug('davam deti roota')
```

Comment:
Let's use English in master (similarly elsewher).
Moreover, I guess this logging is not needed anymore.